spec/controllers/v2/concerns/tailoring_file_spec.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'tailoring_file' do
+  let(:parent) do
+    FactoryBot.create(
+      :v2_policy,
+      :for_tailoring,
+      account: current_user.account,
+      supports_minors: [8]
+    )
+  end
+
+  let(:extra_params) { { policy_id: parent.id, id: item.id } }
+
+  context 'with no tailored rules and no values' do
+    let(:canonical_profile) do
+      FactoryBot.create(
+        :v2_profile,
+        rule_count: 5,
+        value_count: 5,
+        security_guide: FactoryBot.create(:v2_security_guide, rule_count: 3, os_major_version: 9),
+        os_major_version: 9,
+        ref_id_suffix: 'bar',
+        supports_minors: [8]
+      )
+    end
+
+    let(:item) do
+      FactoryBot.create(
```

Comment:
Same here, you can maybe extract the changing params into `let` blocks and define just them across contexts.